docs/detailed_test_config.rst

```diff
@@ -270,3 +270,15 @@ the location of the sandbox directory, the following option can be
 specified in ``$HOME/.teuthology.yaml``::
 
     test_path: <directory>
+
+Shaman options
+==============
+
+Shaman is a helper class which could be used to build the uri for specified
+packages based the 'shaman_host': 'shaman.ceph.com'.
+
+Options::
+
+    use_shaman: True # Enable to use Shaman, False as default
+    shaman:
+    - force_noarch: True # Force to use "noarch" to build the uri
```

Comment:
According this line, the `shaman` includes a list of hashes, but we want `shaman` be a hash.

Reply:
> According to this line, the `shaman` includes a list of hashes, but we want `shaman` be a hash.

@kshtsk 

Sorry, I didn't get it. Could you kindly give an example of it ?

Thanks

Reply:
Okay, just illustrating:

```>>> print(yaml.safe_dump(dict(shaman=dict(force_noarch=True))))
shaman:
  force_noarch: true

>>> print(yaml.safe_dump(dict(shaman=list(dict(force_noarch=True)))))
shaman:
- force_noarch
```

Reply:
But use the code:
```
        self.force_noarch = self.job_config.get("shaman", {}).get("force_noarch", False)
```
Which is supposed to work with first case.

Reply:
@kshtsk 
Okay, good catch. Fixed it, thanks.